lib/ui/views/groups/group_details_view.dart

```diff
@@ -385,3 +396,38 @@ class _GroupDetailsViewState extends State<GroupDetailsView> {
     );
   }
 }
+
+class AddButton extends StatefulWidget {
+  AddButton({@required this.addFunction, this.context, @required Key key})
+      : super(key: key);
+  final Function addFunction;
+  final BuildContext context;
+  @override
+  _AddButtonState createState() => _AddButtonState();
+}
+
+class _AddButtonState extends State<AddButton> {
+  Function dynamicAddFunction;
+  void setAddFunction(bool isActive) {
+    if (isActive == true) {
```

Comment:
You don't need to use equals operator here as bool itself works.

Reply:
Ok, will refactor this in latest commit.